src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+          method="GET"
+          autocomplete="off"
+          data-qa="big">
+
+      <div class="form-search form-search-large">
+        <img src="./images/Search.svg"
+             alt="search"
+             class="icon-large">
+        <input type="search"
+               name="searchLarge"
+               class="search search-large"
+               placeholder="Try &quot;Los Angeles&quot;"
+               data-qa="keypress">
+      </div>
+
+    </form>
+
+    <form action="#"
+          method="GET"
+          autocomplete="off"
+          data-qa="small">
```

Comment:
```suggestion
    <form 
       action="#"
       method="GET"
       autocomplete="off"
       data-qa="small"
     >
```
Форматируй теги вот так